theming.module

```diff
@@ -17,10 +17,13 @@ function theming_theme_registry_alter(&$hooks) {
   ];
   foreach ($override as $name) {
     $hook = &$hooks[$name];
-    $hook['theme path'] = $path;
-    $hook['file'] = $file;
+    // Only replace the theme function when no theme overrides it.
+    if ($hook['function'] == "theme_$name") {
+      $hook['theme path'] = $path;
+      $hook['file'] = $file;
+      $hook['function'] = "theming_$name";
+    }
```

Comment:
Does that mean when there is `my_theme_form_element` it overrides `theming_form_element`? (And before the fix `my_theme_form_element` would have been irgnored?)

Reply:
Exactly. That’s why the `<span>` wasn’t applied.

Reply:
Thanks!